Xamarin.Forms.Platform.iOS/Platform.cs

```diff
@@ -433,6 +432,12 @@ bool PageIsChildOfPlatform(Page page)
 
 		async Task PresentModal(Page modal, bool animated)
 		{
+			AddPageOverlay();
+			while (!_appeared)
+			{
+				await Task.Delay(5);
+			}
```

Comment:
If you want to avoid

`Warning: Attempt to present <Xamarin_Forms_Platform_iOS_ModalWrapper: 0x1647b8c0> on <Xamarin_Forms_Platform_iOS_PlatformRenderer: 0x16117c80> whose view is not in the window hierarchy!`

then the modal should be pushed only after the presenting controller has appeared. We'll resort to `Delay` till the controller has raised `ViewDidAppear`.